docs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "private": true,
-  "name": "docs",
+  "name": "@aws-amplify/ui-docs",
```

Comment:
Do we need to change all the packages' names? For example `docs` is not consumed anywhere.

I thought only changing those under `packages/` is sufficient, isn't it?

Reply:
Fair point. It's not consumed anywhere, but it's also part of `yarn workspaces` and gets installed as part of workspaces.

Reply:
In general though, I think it'd be better to make this change for all packages (see PR description), so that we have a consistent naming standards throughout the monorepo. For example, if someone is creating a new private package, they wouldn't need to worry about how this package is consumed for possible security implications.

Reply:
I think it's preferable to scope them all to be completely safe